examples/app_out_of_tree_controller/src/out_of_tree_controller.c

```diff
@@ -0,0 +1,79 @@
+/**
+ * ,---------,       ____  _ __
+ * |  ,-^-,  |      / __ )(_) /_______________ _____  ___
+ * | (  O  ) |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+ * | / ,--´  |    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+ *    +------`   /_____/_/\__/\___/_/   \__,_/ /___/\___/
+ *
+ * Crazyflie control firmware
+ *
+ * Copyright (C) 2024 Bitcraze AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, in version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * out_of_tree_controller.c - App layer application of an out of tree controller.
+ */
+
+#include <string.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "app.h"
+
+#include "FreeRTOS.h"
+#include "task.h"
+
+// Edit the debug name to get nice debug prints
+#define DEBUG_MODULE "MYCONTROLLER"
+#include "debug.h"
+
+
+// We still need an appMain() function, but we will not really use it. Just let it quietly sleep.
+void appMain() {
+  DEBUG_PRINT("Waiting for activation ...\n");
+
+  while(1) {
+    vTaskDelay(M2T(2000));
+
+    // Remove the DEBUG_PRINT.
```

Comment:
Are these lines still necessary or are the left from some testing?

Reply:
This comes straight from the blog post. I think this has been commented out from the hello world example. I'll remove it.